apps/question-the-docs/backend/app.py

```diff
@@ -23,12 +37,11 @@ def startup_db_client():
         # We wrap our MongoDB to make it a SuperDuperDB!
         _app.superduperdb = superduper(_app.mongodb)
 
-        # EXPLAIN ARTIFACTS HERE.
+        # Artifacts are data that has been pre-processed for AI.
```

Comment:
Aside from the lack of /verb agreement (are/has) 😆 "data" has no meaning.

(Rule of thumb - a definition generally goes "An Artifact is" rather than "Artifacts are".)

Hmmm.... we need a real writeup but let me make a stab at it.

"An Artifact stores the results of a computation, like the training of a model."